samples/quickstart.js

```diff
@@ -16,6 +16,59 @@
 'use strict';
 
 // [START automl_quickstart]
-// TBD
-// [END automl_quickstart]
+async function main(
+  projectId,
+  computeRegion,
+  modelId,
+  filePath,
+  scoreThreshold
+) {
```

Comment:
I just pulled this quickstart out from one of the samples directories; better than "TBD".